scripts/load_modules_post.sh

```diff
@@ -7,12 +7,10 @@
 # provided by the STM32H7.
 
 modprobe industrialio
-insmod /usr/arduino/extra/x8h7_drv.ko
 insmod /usr/arduino/extra/x8h7_can.ko
 insmod /usr/arduino/extra/x8h7_gpio.ko
 insmod /usr/arduino/extra/x8h7_adc.ko
 insmod /usr/arduino/extra/x8h7_rtc.ko
 insmod /usr/arduino/extra/x8h7_pwm.ko
 insmod /usr/arduino/extra/x8h7_uart.ko
-insmod /usr/arduino/extra/x8h7_ui.ko
-insmod /usr/arduino/extra/x8h7_h7.ko
+insmod /usr/arduino/extra/x8h7_ui.ko
```

Comment:
No newline :cry: 

:wink:

Reply:
I should have fixed it from 134622e72103b6d67c6c92a190d6d71f60aa9ad5